fdbrpc/protos/file_transfer.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package fdbrpc;
+
+service FileTransferService {
+    rpc DownloadFile(DownloadRequest) returns (stream DownloadChunk);
```

Comment:
Should the method be transfer since this is a filestransfer service rather than DownloadFile?

Reply:
Felt transfer sometimes implies moving and download felt more explicit. But open to changing this name or of the module.

Reply:
Just a nit on my part. Wonder what others do regards naming in their 'transfer' service.

Reply:
REST file transfer uses upload/download for API methods.